dev/benchmarks/macrobenchmarks/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -285,7 +285,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"$FLUTTER_ROOT\"/packages/flutter_tools/bin/macos_assemble.sh\ntouch Flutter/ephemeral/tripwire\n";
+			shellScript = "\"$FLUTTER_ROOT\"/packages/flutter_tools/bin/macos_assemble.sh && touch Flutter/ephemeral/tripwire\n";
```

Comment:
This matches the latest macOS template:

https://github.com/flutter/flutter/blob/bf6481104dc225ae3c13329088125c70280732ae/packages/flutter_tools/templates/app/macos.tmpl/Runner.xcodeproj/project.pbxproj.tmpl#L346

Without this, the Xcode build succeeds even if the macOS assemble step fails.